docs/streams/developer-guide/config-streams.html

```diff
@@ -257,7 +258,12 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
             <td colspan="2">The maximum number of records to buffer per partition.</td>
             <td>1000</td>
           </tr>
-          <tr class="row-even"><td>cache.max.bytes.buffering</td>
+          <tr class="row-even"><td>cache.max.bytes</td>
+            <td>Medium</td>
+            <td colspan="2">Maximum number of memory bytes to be used for record caches across all threads. Note that at the debug level you can use cache.size to monitor the actual size of the Kafka Streams cache.</td>
+            <td>10485760 bytes</td>
```

Comment:
```suggestion
            <td>10485760</td>
```

Reply:
The description already say `Maximum number of memory bytes` so seems we can drop `bytes` here?